nbs/models.timesnet.ipynb

```diff
@@ -439,14 +439,12 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.                     #loss=MQLoss(quantiles=[0.2, 0.5, 0.8]),
</pre><p>Let's remove this comments, just for a cleaner documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>